### ISSUE TITLE
Security 100 Labs: Create a Data Bunker Account: General improvement ideas

### ISSUE THREAD

**Issue by mnylen** [awslabs/aws-well-architected-labs#40]
The Create a Data Bunker chapter seems to assume knowledge that hasn't been covered previously in the labs:

1. The prerequisities assume you have access to the root account for an AWS Organization. However, creating the organization hasn't been covered in the labs previously to my knowledge.

2. In Step 1 of the labs we create a new security account. However, it is not explained how to gain access to that account after creation. That requires the IAM user to be modified with a permission to perform sts:AssumeRole on the arn:aws:iam:SECURITYACCOUNTID:role/OrganizationAccountAccessRole and the user then needs use the Switch Role functionality in the console to switch to this account.

    This should be instructed somehow.

3. In Step 3 the labs instruct readers to modify OrganizationAccountAccessRole to only have read-only privileges to the security account. This raised some further questions:

    - As no additional IAM roles or IAM users were created in this guide for the security account, what are the best practices for allowing future administrative access to the security account? Surely there are scenarios where this is needed. E.g. later allowing also VPC Flow Logs to be delivered to the same bucket, or modifying the retention period. Or would a best practice be to create another security account for this purpose?

        As far as I know, if the reader blindly follows the guide, the only way to gain admin access back is to request password recovery for the email address you entered when creating the security account 

    - When you create a security account, should you actually immediately following that request a password via password reset and setup MFA on the security account root credentials? Otherwise your security account's security is only as good as the security of your email system.

**Comment by ByronPogson** [comment]
@mnylen Thanks heaps for your feedback on this lab! I originally created this in context to a larger Quest - Quick steps to security success and there are clearly some gaps in running this as a stand alone lab. I've added more working in around what the intention of each step is and noting that for a lot of the steps it's going to depend largely on your organization's policy, the steps that I've outlined here are suggested guidance if you're building from scratch.

In regards to each of your points specifically:
1. I have updated the wording of the prerequisites to be clearer that a root organizations account is required ahead of creating the lab. See the ‘Quick Steps to Security Success’ for a discussion on usage of organizations
2. I've added in an explicit step to switch into the security account.
3. I've changed this step to optional and focused more on _why_ this step is important. I've called out that human access should be break glass only and added some links to discussions of best practices.
